.github/workflows/pr-labels.yaml

```
@@ -0,0 +1,27 @@
+---
+name: PR Labels
+
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+    types:
+      - opened
+      - labeled
+      - unlabeled
+      - synchronize
+  workflow_call:
```

```suggestion
    types:
      - labeled
      - unlabeled
```
These are not needed here